app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
The use of `command.split()` is correct here, but ensure that you do not repeat this operation unnecessarily elsewhere in the code. According to the checklist, each necessary method should be called only once.